### PR TITLE
make_openbazaar.sh typo fix for bash and fish shells, remove brew call and added wine as required dep in README.md

### PR DESCRIPTION
**Pull request by M0Rf30**
fish says
Failed to execute process './make_openbazaar.sh'. Reason:

exec: Exec format error

The file './make_openbazaar.sh' is marked as an executable but could not be run by the operating system.

bash says;
make_openbazaar.sh: riga 1: #!/bin/sh: File o directory non esistente

### CODE REVIEW

**Comment by M0Rf30** [comment]
Refactor make_openbazaar.sh in order to be distro agnostic

**Comment by M0Rf30** [comment]
Perhaps there will be some errors for OSX building process, but I can't test

**Comment by M0Rf30** [comment]
Instead of accept my pull request and fix macos errors compilation after, you think could be better read the suggestions and make my fork unusable for merge?

**Comment by hoffmabc** [comment]
Sorry if I am misunderstanding you @M0Rf30 ...I don't understand the question. I am going to read through and try to help resolve.

File: make_openbazaar.sh [hunk 1]
```diff
@@ -1,38 +1,34 @@
-﻿﻿#!/bin/sh
+#!/bin/bash
```
Comment (hoffmabc) [inline make_openbazaar.sh:1]: Why not just use the system's shell. Are there things here that are Bash specific? And in most cases sh will resolve to bash anyways.

**Comment by hoffmabc** [comment]
OK I think I understand that you are agitated because I broke your PR with some later commits. I didn't realize you even had this PR submitted since it was a few days ago and I wasn't around. Apologies. I will try and pull down your branch and figure out how to clean it up for you.

**Comment by hoffmabc** [comment]
OK I resolved the conflicts and merged. Sorry for being confusing.

**Comment by M0Rf30** [comment]
You are a sir, I'm happy to contribute to this futuristic project. My tones were not dictated by anger or ambition, but truly spirit of partecipation.

**Comment by M0Rf30** [comment]
Could you take a look at this?
https://aur.archlinux.org/packages/?O=0&K=openbazaar

This is my work of packaging for ArchLinux AUR. All the variants are fully working since your first works
Archlinux has a different method of packaging, more distro-agnostic respect to the most popular distros.

**Comment by yagoulas** [comment]
Hello, there is something really weird with this pull request. Can you clarify that all these commits were meant to be submitted?

**Comment by M0Rf30** [comment]
I'm cleaning some errors regarding the packaging and extra data not needed by a well-done package. I've just collaborated to rewrite the make_openbazaar script. Now I'm just refining my work. The commits speak clearly

**Comment by yagoulas** [comment]
Can you recheck the list of the commits that were merged? The list is huge and you have several changes that are related to windows and you don't mention them anywhere. The list of commits: https://github.com/OpenBazaar/OpenBazaar-Installer/pull/7/commits

**Comment by M0Rf30** [comment]
for ob.nsi:
I've basically cleaned nodejs refs not required
setted PYTHON global variables in win environment
setted python.exe in win firewall in order to obtain first runnable openbazaar session
removed an unneeded variable in nsis files
splitted win64 and win32 packages
used noupnp branch for server, cause of miniupnpc not compiled yet on win OS
used upx compression properly
used correct python executable in systray
changed pyinstaller to git release cause of "match_ssl_hostname" error

**Comment by yagoulas** [comment]
Can you join slack to discuss this?

**Comment by M0Rf30** [comment]
How. I've never used slack. Invitation needed?

**Comment by hoffmabc** [comment]
@M0Rf30 go here for an invite. https://openbazaar-slackin-drwasho.herokuapp.com/

**Comment by M0Rf30** [comment]
I'm on slack

**Pull request merged**